client/app/bundles/KudosApp/components/KudosList.jsx

```diff
@@ -1,32 +1,41 @@
-import React, { PropTypes } from 'react';
-import TabBarContainer from '../containers/TabBarContainer';
-import Kudo from './Kudo';
-import _ from 'lodash';
-import moment from 'moment';
-import injectTapEventPlugin from 'react-tap-event-plugin';
+import React, { PropTypes } from 'react'
+import TabBarContainer from '../containers/TabBarContainer'
+import Kudo from './Kudo'
+import _ from 'lodash'
+import moment from 'moment'
+import injectTapEventPlugin from 'react-tap-event-plugin'
+import { COLOR_CLASSES } from '../constants/appConstants'
+
+const { TEAL, GREEN, ORANGE } = COLOR_CLASSES
+const color_classes = [TEAL, GREEN, ORANGE]
 
 injectTapEventPlugin();
 
-const List = ({ giverId, kudos, likeKudo, unlikeKudo, updateKudo }) => {
+const List = ({ userId, kudos, likeKudo, unlikeKudo, updateKudo }) => {
+  const numKudos = kudos.length
+  const numColors = color_classes.length
+
   return <div className="kudos-list">
-    { kudos.length > 0 ? (
-      kudos.map(kudo => <
-        Kudo id={kudo.id}
-        giverId={giverId}
-        key={kudo.id}
-        kudo={kudo}
-        likeKudo={likeKudo}
-        unlikeKudo={unlikeKudo}
-        updateKudo={updateKudo}
-        />)
-    ) : (
-      'No kudos'
-    )}
+    { numKudos > 0 ? (
+      kudos.map((kudo, index) => (
+        <Kudo
+          id={ kudo.id }
+          color_class={ color_classes[(numKudos-index)%numColors] }
```

Comment:
FYI I did this so that when a new Kudo is submitted (and inserted at the top AKA becomes the first item in the array) this won't affect the colors of the existing Kudos on the screen.

Right now it works, but after adding some kind of scroll to load more kudos, we'll need a different method of assigning colors since kudos can be appended at the beginning and end

Reply:
Thanks! I'll keep this is mind as I try to add scroll load feature next